x-pack/plugins/canvas/public/state/middleware/aeroelastic.js

```diff
@@ -21,7 +23,9 @@ import { selectElement } from '../actions/transient';
 import { addPage, removePage, duplicatePage } from '../actions/pages';
 import { appReady } from '../actions/app';
 import { setWorkpad } from '../actions/workpad';
-import { getElements, getPages, getSelectedPage, getSelectedElement } from '../selectors/workpad';
+import { getNodes, getPages, getSelectedPage, getSelectedElement } from '../selectors/workpad';
+
+const isGroupId = id => id.startsWith('group_');
```

Comment:
Can group ids follow the same convention of `type-uuid` we've been using else were?  

Elements current look like `element-f8ae2ac4-1bd8-4721-98a9-8827aa1ea36f` whereas groups are `group_845053003004`

Reply:
Pushing the change for this soon, it was a bit trickier as page cloning, structural copy/paste have to map to new id values and the logic changed (previously we just glued an extension, but now it'll be uniform too). Looking forward to your decision on the above feedback item ^^